packages/x-components/src/x-modules/related-tags/components/related-tags.vue

```diff
@@ -71,6 +77,14 @@
     @Prop()
     protected maxItemsToRender?: number;
 
+    /**
+     * Flag to indicate if the curated tags should be displayed different.
+     *
+     * @public
+     */
+    @Prop({ default: false, type: Boolean })
+    protected shouldHighlightCurated!: boolean;
```

Comment:
Why did you rename this? Doesn't it make sense to name  RelatedTag and RelatedTags highlight prop equal? `highlightCurated`

Reply:
Because you told me 😆

Reply:
My English is really bad.